src/Microsoft.DotNet.Build.Tasks/Targets/toolruntime.targets

```diff
@@ -14,8 +14,10 @@
     <ToolHost Condition="'$(OS)' != 'Windows_NT'">corerun</ToolHost>
 
     <ToolTargetFramework Condition="'$(ToolTargetFramework)' == ''">DNXCore,Version=v5.0</ToolTargetFramework>
+    <ToolOS Condition="'$(OS)' == 'Windows_NT'">win7</ToolOS>
+    <ToolOS Condition="'$(OS)' != 'Windows_NT'">ubuntu.14.04</ToolOS>
```

Comment:
I am not sure.  I decided against doing something like this in #293 since not only is it wrong for different distros, it's also wrong for different non windows oses (e.g OSX).  I think no matter the default, the build consuming build tools is going to need to setup ToolNugetRuntimeId, there really isn't a good default.  I sort of wish we would just always restore the windows version so folks that need to set it discover it quickly rather than it working if they are on ubuntu and blowing up everywhere else?